docs/src/main/sphinx/release/release-478.md

```diff
@@ -0,0 +1,140 @@
+# Release 478 (dd Oct 2025)
+
+## General
+
+* Add support for column lineage in `UNNEST` clause. ({issue}`16946`)
+* Add `allowed-retry-policies` configuration property to specify which retry
+  policies can be selected by user. ({issue}`26628`)
+* Add support for loading plugins from multiple directories. ({issue}`26855`)
+* Add `/v1/integrations/gateway` endpoint for integration with Trino Gateway. ({issue}`26548`)
+* Allow to drop catalog that failed to load correctly. ({issue}`26918`)
+* Improve performance of queries with `ORDER BY`. ({issue}`26725`)
+* Improve performance of `MERGE` statement involving `NOT MATCHED` case. ({issue}`26759`)
+* Improve performance of spilling join queries. ({issue}`26076`)
+* Fix potential incorrect results when reading `row` type. ({issue}`26806`)
+* Make `metadata.catalogs` table return all catalogs. ({issue}`26918`)
+* Ensure that queries with and without `EXPLAIN ANALYZE` are planned in the same
+  way. ({issue}`26938`)
+
+## Security
+
+* Propagate `queryId` to [Open Policy Agent](/security/opa-access-control)
+  authorizer. ({issue}`26851`)
+
+## Web UI
```

Comment:
Also ... not sure by @martint did not want changes for preview UI in the release notes in the past .. however I think we should have them

Reply:
I don't agree with that reasoning at all. We're shipping the preview web UI because we want users to use that and retire the old UI. So it doesn't make sense to hide away the developments happening on that front. Also, the UI being new or not having all the features of the old one doesn't make it "not suitable for production usage". It's easy to toggle back and forth between new and old UI with a button after https://github.com/trinodb/trino/pull/26705 and `web-ui.preview.enabled` is enabled by default.